ROADMAP.md

```diff
@@ -1,40 +1,67 @@
+  * Update promotion and release channels to align with storage provider specific statuses [#1885](https://github.com/rook/rook/issues/1885)
+  * Refactor test framework and helpers to support multiple storage providers [#1788](https://github.com/rook/rook/issues/1788)
+  * Isolate and parallelize storage provider testing [#1218](https://github.com/rook/rook/issues/1218)
+  * Longhaul testing pipeline [#1847](https://github.com/rook/rook/issues/1847)
+* Custom resource validation, progress, status [#1539](https://github.com/rook/rook/issues/1539)
+* Design for Volume Snapshotting and policies (consider aligning with SIG-storage) [#1552](https://github.com/rook/rook/issues/1552)
+* Support for dynamic provisioning of new storage types
+  * Dynamic bucket provisioning [#1705](https://github.com/rook/rook/issues/1705)
+  * Dynamic database provisioning [1704](https://github.com/rook/rook/issues/1704)
+* New storage providers
+  * NFS operator and CRDs (backed by arbitrary PVs) [#1551](https://github.com/rook/rook/issues/1551)
+  * Cassandra design [#1910](https://github.com/rook/rook/issues/1910)
+* CockroachDB
+  * Secure deployment using certificates [#1809](https://github.com/rook/rook/issues/1809)
+  * Helm chart deployment [#1810](https://github.com/rook/rook/issues/1810)
+  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
+* Minio
+  * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
+  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
+* Ceph
+  * Update to Mimic [#1536](https://github.com/rook/rook/issues/1536)
+  * Automated upgrade support (initial) [#997](https://github.com/rook/rook/issues/997)
+  * Manage an existing Ceph cluster (basic) [#1868](https://github.com/rook/rook/issues/1868)
+  * OSDs
+    * Run on arbitrary PVs (local storage) as an alternative to host path [#796](https://github.com/rook/rook/issues/796) [#919](https://github.com/rook/rook/issues/919)
+    * Minimize or eliminate running with privileged containers (e.g., reduced access to /dev) [#1944](https://github.com/rook/rook/issues/1944)
+    * Disk management (adding, removing, and replacing disks) [#1435](https://github.com/rook/rook/issues/1435)
+    * ceph-volume provisioning [#1342](https://github.com/rook/rook/issues/1342)
+  * Mgr and plugins
+    * Placement group balancer support (enable the mgr module)
+  * File
+    * NFS Ganesha CRD [#1799](https://github.com/rook/rook/issues/1799)
+    * Dynamic Volume Provisioning for CephFS [#1125](https://github.com/rook/rook/issues/1125)
+  * Object
+    * Multi-site configuration [#1584](https://github.com/rook/rook/issues/1584)
+    * CRD for object store users [#1583](https://github.com/rook/rook/issues/1583)
 
 ## Rook 1.0
 
-* Declare numerous types as v1
-* Automated upgrades
+* Declare Ceph to be stable v1
```

Comment:
I think adding `CRDs` after Ceph would make it 100% clear what we are talking about.